Anders-Engman/BegsList
# Add actual auth

added auth screens to main page and added basic modal functionality

to test: 
in main route, click on Log-in or Sign up to see the modals pop and the background obscured

hit escape to clear the modal.
